INTERVIEWS.md

```diff
@@ -7,6 +7,8 @@ We usually run more than one interview for each candidate.
 We also send them an introductory email presenting our company, our projects and our way of work.
 When running your interview, you should be aware if the interview is **technical** (run by developers) or **non-technical** (performed by our Tech Lead or Country Manager).
```

Comment:
I found 2 misspelled words in this sentence: 
- **non-technical**
- **technical**